Makefile

```diff
@@ -931,6 +931,14 @@ test-go-unit-tbot:
 		| tee $(TEST_LOG_DIR)/unit.json \
 		| gotestsum --raw-command -- cat
 
+# Runs CLI reference doc generator unit tests
+.PHONY: test-go-unit-clidocs
+test-go-unit-clidocs: FLAGS ?= -shuffle on -tags docs -run 'TestUpdateAppUsageTemplate'
+test-go-unit-clidocs:
+	go test -json $(FLAGS) ./lib/utils/... \
+		| tee $(TEST_LOG_DIR)/unit.json \
+		| gotestsum --raw-command -- cat
+
```

Comment:
Do we need a separate target for this, or can we just pass the `docs` tag to the existing unit test target?